galsim/table.py

```diff
@@ -165,18 +179,22 @@ def __call__(self, x):
             if dimen > 2:
                 raise ValueError("Arrays with dimension larger than 2 not allowed!")
             elif dimen == 2:
-                f = np.zeros_like(x)
-                for i in xrange(x.shape[0]):
-                    f[i,:] = np.fromiter((self.table(float(q)) for q in x[i,:]), dtype='float')
+                f = np.empty_like(x.flatten(), dtype=float)
```

Comment:
Thanks Josh.  I hadn't appreciated that distinction.  I had thought `flatten` did what it turns out `ravel` actually does, which is return a view if possible and a copy if necessary to make it contiguous.  I'll need to get in the habit of writing `ravel` whenever my brain thinks `flatten`.  :)